src/Datadog.Trace/ScopeManager.cs

```diff
@@ -0,0 +1,12 @@
+namespace Datadog.Trace
+{
+    internal class ScopeManager : IScopeManager
+    {
+        public Scope Active => DatadogScopeStack.Active;
```

Comment:
Having the static reference wrapped in a scope manager will satisfy dependency injection scenarios and will make the code testable. (Good for users, good for us)